src/postamble.js

```diff
@@ -425,7 +425,11 @@ function exit(status, implicit) {
       throw 'unwind';
     } else {
 #if PTHREADS_DEBUG
+#if EXIT_RUNTIME
       err('main thread called exit: keepRuntimeAlive=' + keepRuntimeAlive() + ' (counter=' + runtimeKeepaliveCounter + ')');
+#else
+      err('main thread called exit: keepRuntimeAlive=' + keepRuntimeAlive());
+#endif
```

Comment:
Is this change related?

Reply:
Sorry, no, this just just came up when I was trying to enable PTHREAD_DEBUG in a build without EXIT_RUNTIME.

Do you think its worth splitting out?

Reply:
No, that's fine. Just making sure I'm not misunderstanding anything.